src/Features/Core/Portable/ExternalAccess/UnitTesting/API/INewUnitTestingIncrementalAnalyzerImplementation.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal interface INewUnitTestingIncrementalAnalyzerImplementation
+    {
+#if false // Not used in unit testing crawling
+        // Task NewSolutionSnapshotAsync(Solution solution, CancellationToken cancellationToken);
+        // Task DocumentOpenAsync(Document document, CancellationToken cancellationToken);
+        // Task DocumentCloseAsync(Document document, CancellationToken cancellationToken);
+        // Task DocumentResetAsync(Document document, CancellationToken cancellationToken);
+        // void RemoveProject(ProjectId projectId);
+
+        // [Obsolete]
+        // bool NeedsReanalysisOnOptionChanged(object sender, UnitTestingOptionChangedEventArgsWrapper e);
+#endif
```

Comment:
this is based on http://index/?leftProject=Microsoft.CodeAnalysis.Workspaces&leftSymbol=s2uswntn4lf6&file=ExternalAccess%5CUnitTesting%5CUnitTestingIncrementalAnalyzer.cs&line=15 and seeing what methods unit-testing actually has real logic for.